scene_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/veandco/go-sdl2/sdl"
+	"testing"
+)
+
+func TestNewSceneCreated(t *testing.T) {
```

Comment:
Is there a way in Go to check in tests if the object is instance of some struct? 
I've tried `ok := object.(someStruct)`, but seems like it doesn't work

Reply:
Something like this:
```go
func assert(argument interface{}) {
	obj, ok := argument.(renderableObjectInterface)
	if !ok {
		panic("foo")
	}
	obj.fooMethod()
}

func assert2(argument renderableObjectInterface) {
	obj, ok := argument.(renderableObjectInterface)
	if !ok {
		panic("foo")
	}
	obj.fooMethod()
}
```

Reply:
So it works only with interfaces, but doesn't work with a struct, I see now. Thanks!

Reply:
Its not working?
```go
type imAnObject struct {
}

func (i imAnObject) fooMethod() string {
	return "bar"
}

func isAnObject(argument interface{}) bool {
	_, ok := argument.(imAnObject)
	return ok
}

func printResult(result bool) {
	if result {
		fmt.Println("Yes is object")
	} else {
		fmt.Println("Not Object")

	}
}

func TestAssert(t *testing.T) {
	t.Run("fail with a string", func(t *testing.T) {
		printResult(isAnObject("holi"))
	})

	t.Run("fail with a pointer of object", func(t *testing.T) {
		printResult(isAnObject(&imAnObject{}))
	})

	t.Run("OK passing the object", func(t *testing.T) {
		printResult(isAnObject(imAnObject{}))
	})
}

```

```
=== RUN   TestAssert
=== RUN   TestAssert/fail_with_a_string
Not Object
=== RUN   TestAssert/fail_with_a_pointer_of_object
Not Object
=== RUN   TestAssert/OK_passing_the_object
Yes is object
--- PASS: TestAssert (0.00s)
    --- PASS: TestAssert/fail_with_a_string (0.00s)
    --- PASS: TestAssert/fail_with_a_pointer_of_object (0.00s)
    --- PASS: TestAssert/OK_passing_the_object (0.00s)
PASS
```

Reply:
Thank you a lot, that's very helpful. This is exactly what I lacked!